Berlin/BerHsOr9834.xml

```diff
@@ -70,10 +70,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
                </physDesc>
                <history>
                    <origin>
-                      <origPlace> <placeName ref="LOC2422DagaE"/> </origPlace> 
+                      <origPlace> <placeName ref="LOC2422DagaEs"/> Dāgā ʾƎsṭifānos </origPlace> 
                       <origDate notBefore="1300" notAfter="1600"/>
                    </origin>
-                   <provenance>The manuscript was private posession of <persName ref="PRS5072Hammersc"/>. He had received it as a gift from a local monk in <placeName ref="LOC2422DagaE"/> in <date when="1968">1968</date>.
+                  <provenance>The manuscript was private posession of <persName ref="PRS5072Hammersc"/>. He had received it as a gift from a local monk in Dāgā ʾƎsṭifānos <placeName ref="LOC2422DagaEs"/> in <date when="1968">1968</date>.
```

Comment:
This encoding `(Dāgā ʾƎsṭifānos <placeName ref="LOC2422DagaEs"/>)` will lead to double printing of the name. Simply  `<placeName ref="LOC2422DagaEs"/>` is enough. If you are not happy with how the name is rendered, and want to make sure that your preferred version is printed, you can write  `<placeName ref="LOC2422DagaEs">Dāgā ʾƎsṭifānos</placeName>` instead.

Reply:
ah no, now I understand, the name was not printed at all because the pointer had a typo, which I understand only now. OK
will update, thank you